### PR TITLE
#63

### PR DESCRIPTION
**Pull request by zhiweigan**

### CODE REVIEW

File: fluid-music/src/fluid/plugin-tequaliser.js [hunk 1]
```diff
@@ -73,7 +73,23 @@ const pluginTEqualiser = {
...
+   * @param {number} value a boolean to set the parameter to
+   * @param {number} [timeInWholeNotes] time to insert automation point in
+   *    quarter notes. If no time is supplied, set the initial value
+   * @param {number} [curve=0] A number from [-1, 1] (inclusive), which
+   *    represents the curvature of the line formed by this point and the next
+   *    point. Zero implies a linear change. Higher values create a curve that
+   *    begins slowly and accelerates. Lower values create a curve that begins
+   *    quickly, and decelerates.
+   */
+  setBand1Active(value, timeInWholeNotes, curve) {
```
[the remark this answers is not in the document]
Comment (CharlesHolbrow) [reply fluid-music/src/fluid/plugin-tequaliser.js:89]: Oh actually, that looks ok, I think I was looking at the wrong thing.

[the remark this answers is not in the document]
Comment (zhiweigan) [reply fluid-music/src/fluid/plugin-tequaliser.js:89]: Gotcha, do you still want me to make any changes?

[the remark this answers is not in the document]
Comment (CharlesHolbrow) [reply fluid-music/src/fluid/plugin-tequaliser.js:89]: Hmm, Yes I think it's worth adding the default value, which allows the consumer to write `setBand1Active()` which is clean code. Thank you!

[the remark this answers is not in the document]
Comment (zhiweigan) [reply fluid-music/src/fluid/plugin-tequaliser.js]: Done!

**Pull request merged**